app/code/Magento/InventorySales/Plugin/StockState/CheckQuoteItemQtyPlugin.php

```diff
@@ -163,4 +164,25 @@ private function getNumber($qty)
 
         return $qty;
     }
+
+    /**
+     * Returns websiteId assigned to store when scopeId is passed
+     * Returns default websiteId if no scopeId is passed
+     * @param int $scopeId
+     * @return string
+     */
+    private function resolveWebsiteId($scopeId = null) {
```

Comment:
Might as well return code here, as `websiteId` is converted to code in the next line.

Reply:
Changed in newest commit